#
# Event-Out port the message pertains to:
#
# event_out_mark_t - enumeration
uint8 value

#Pre-Defined ENUM Constants
	uint8 EVENTOUT1    = 1 #  [g080]
	uint8 EVENTOUT2    = 2 #  [g080]
	uint8 EVENTOUT3    = 3 #  [g080]
	uint8 EVENTOUT4    = 4 #  [g080]
	uint8 EVENTOUT5    = 5 #  [reserved]
	uint8 EVENTOUT6    = 6 #  [reserved]
	uint8 EVENTOUT7    = 7 #  [reserved]
	uint8 EVENTOUT8    = 8 #  [reserved]
	uint8 EVENTOUT_PPS = 31 #  GNSS Receiver PPS  [g080]
